topics/reverse-engineering/index.md

```diff
@@ -0,0 +1,9 @@
+---

```

Comment:
```suggestion
---
```

Reply:
there is something going on with the characters on this line

Reply:
I'm on wsl. Performed a `dos2unix` on `index.md`. 
No error localy with `script/cibuild`
We will see result of CI pipeline !

Reply:
All tests passed

Reply:
All tests passed.